test/passes/fannkuch3.bin.txt

```diff
@@ -2567,110 +2567,110 @@ Abbrev table for offset: 0x00000000
 
 0x000000c3:     DW_TAG_variable [13]  
                   DW_AT_location [DW_FORM_sec_offset]	(0x0000001d: 
-                     [0xffffffff,  0x00000006): 
-                     [0x00000007,  0x00000030): DW_OP_consts +0, DW_OP_stack_value
-                     [0x00000046,  0x0000004b): DW_OP_WASM_location 0x0 +1, DW_OP_stack_value
-                     [0x000000dc,  0x000000e5): DW_OP_consts +1, DW_OP_stack_value
-                     [0x00000121,  0x0000012b): DW_OP_WASM_location 0x0 +0, DW_OP_stack_value
-                     [0x00000169,  0x00000176): DW_OP_consts +0, DW_OP_stack_value
-                     [0x00000258,  0x00000263): DW_OP_consts +0, DW_OP_stack_value
-                     [0x00000269,  0x00000272): DW_OP_consts +1, DW_OP_stack_value
-                     [0x000002ae,  0x000002b8): DW_OP_WASM_location 0x0 +0, DW_OP_stack_value
-                     [0x000002f6,  0x00000303): DW_OP_consts +0, DW_OP_stack_value)
+                     [0xffffffff,  0x0000000d): 
+                     [0x00000000,  0x00000029): DW_OP_consts +0, DW_OP_stack_value
```

Comment:
This style of test is convenient to detect regressions, but one thing I don't like about it is that it's basically impossible to tell in this context whether the updated test results are what we actually want

Reply:
Yeah, that's a downside. But I skimmed all the output to see that there are no changes of valid => 1 (the invalid value) or invalid => valid, and I checked a bunch of non-invalid values in detail and they all looked ok.